packages/graphql/mock-server/README.md

```diff
@@ -0,0 +1,171 @@
+- `localMocks` - a list of local GraphQL schemas to define your own GraphQL schemas and resolvers or extend an exisiting local or remote schema.
+
+```json
+{
+  "hops": {
+    "remoteSchemas": [
+      {
+        "url": "https://api.github.com/graphql",
+        "headers": {
+          "Authorization": "bearer [GITHUB_API_TOKEN]"
+        }
+      }
+    ],
+    "localMocks": [
+      "<rootDir>/mocks/feature1",
+      "<rootDir>/mocks/feature2",
+      "<rootDir>/mocks/feature3"
+    ]
+  }
+}
+```
+
+## GraphQL Mock-Server usage examples
+
+- [Define your own schema extentions](#define-your-own-schema-extentions)
+- [Use a remote schema as a proxy](#use-a-remote-schema-as-a-proxy)
+- [Overwrite remote data using custom resolver](#overwrite-remote-data-using-custom-resolver)
+- [Extend a schema and attach custom resolver](#extend-a-schema-and-attach-custom-resolver)
+- [Define and mock scalar, interface, union and enum types](#define-and-mock-scalar-interface-union-and-enum-types)
+- [Define and mock mutations with success and error fields](#define-and-mock-mutations-with-success-and-error-fields)
+- [Define and mock cursor-based pagination with connection and edges](#define-and-mock-cursor-based-pagination-with-connection-and-edges)
+
+## Define your own schema extentions
+
+To define your own schema extentions it is necessary to stick to a few conventions. Every schema extention should be summarized to a closed feature based bundle in shape of a folder. In this case we call it feature. Every feature folder can contain
+
+- `schema.graphql` - to crate or extend a schema using the schema definition language [SDL](https://facebook.github.io/graphql/)
+- `resolvers.js` - to implement dataset resolvers using NodeJS
+- `mock.js` - to implement mock datasets based on types
+
+It's required to configure your schema extentions and dataset resolvers in the correct order. The reason for this is quite simple. Sometimes a type depends on a type from a different schema file or on a type from a remote schema.
+
+**`package.json`**
+
+```json
+{
+  "hops": {
+    "graphqlMockConfigFile": "<rootDir>/graphql-mock-config.js"
+  }
+}
+```
+
+**`graphql-mock-config.js`**
+
+```javascript
+module.exports = {
+  remoteSchemas: [],
+  schemaMocks: ['./mocks/feature1', './mocks/feature2', './mocks/feature3'],
+};
+```
+
+## Use a remote schema as a proxy
+
+You can configure one or more GraphQL server URLs to merge them together to a larger schema (also known as [schema stitching](https://www.apollographql.com/docs/graphql-tools/schema-stitching.html)).
+
+**`package.json`**
+
+```json
+{
+  "hops": {
+    "graphqlMockConfigFile": "<rootDir>/graphql-mock-config.js"
+  }
+}
+```
+
+**`graphql-mock-config.js`**
+
+````javascript
+module.exports = {
+  remoteSchemas: [
+    {
+      url: 'https://api.github.com/graphql',
+      headers: {
+        Authorization: 'bearer <YOUR GITHUB API TOKEN>',
```

Comment:
Same here - I suppose you could just concat the string with `process.env.GITHUB_API_TOKEN`.